store/sqlstore/upgrade.go

```diff
@@ -651,6 +652,29 @@ func UpgradeDatabaseToVersion511(sqlStore SqlStore) {
 	// TODO: Uncomment following condition when version 5.11.0 is released
 	// if shouldPerformUpgrade(sqlStore, VERSION_5_10_0, VERSION_5_11_0) {
 
+	// Enforce all teams have an InviteID set
+	res := <-sqlStore.Team().GetAll()
```

Comment:
Can we do a custom select here that only grabs blank inviteids? In the 99% case there shouldn't be any teams without inviteIDs.